lib/gui/app/components/modal/services/modal.js

```diff
@@ -43,13 +43,11 @@ module.exports = function ($uibModal, $q) {
       size: 'sm'
     })
 
-    analytics.logEvent('Open modal', {
-      template: options.template
-    })
+    analytics.logEvent('Open modal')
```

Comment:
Hmmm, so this logEvent will no longer show _which_ modal was opened?

Reply:
`options.template` used to be a local URL, but its now inline HTML

Reply:
Ahh, I see. I'm wondering how useful a `'Open modal'` logEvent will be if we can't even tell which modal was opened? :-/

Reply:
Fair enough. I added a `name` option to the `ModalService`, so we can track what it is.